bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md

```diff
@@ -0,0 +1,915 @@
+
+Aaron van Wirdum: 00:07:55
+
+Yeah, I think especially episode 17. Yeah, both 17 and 18, I think most of this specific change was in episode 17. So this was included in Bitcoin Core 21 as well. So Eclipse attacks are now harder to pull off against Bitcoin Core 21 nodes.
+
+Sjors Provost: 00:08:15
+
+Yes.
+
+Aaron van Wirdum: 00:08:16
+
+Nice.
+
+Sjors Provost: 00:08:16
+
+All right.
+
+## BIP-157 "Neutrino Filters"
+
+Aaron van Wirdum: 00:08:17
+
+Next point.
+
+Sjors Provost: 00:08:18
+
+The next point is BIP-157, also known as neutrino filters. We have not talked about that yet, so we can briefly explain it.There are lots of mobile wallets out there, especially for Lightning, that don't want to download the entire blockchain.
+
+Aaron van Wirdum: 00:08:34
+
+Yeah, or just Lite wallets in general. They don't have to be mobile, just something that doesn't download and verify the entire blockchain.
+
+Sjors Provost: 00:08:41
+
+Yeah, though I think the number of excuses you have to run a Lite wallet outside of a mobile situation are reduced.
+
+Aaron van Wirdum: 00:08:48
+
+I mean, a lot of people don't feel like waiting for two days for the whole thing to sync.
+
+Sjors Provost: 00:08:53
+
+Yeah, that's true for the initial experience.
+
+Aaron van Wirdum: 00:08:55
+
+I mean it's a bad idea or you know definitely not preferable but that would be the reason.
+
+Sjors Provost: 00:09:01
+
+Yeah, so what these filters do is they instead of you instead of downloading every block you download these filters which are much smaller than a block and then you can take your own list of addresses and run your own list of addresses against that filter And the filter will tell you whether or not there's something in the block pertaining to your addresses.
+
+Aaron van Wirdum: 00:09:20
+
+Yeah, or at least possibly.
+
+Sjors Provost: 00:09:22
+
+Yeah, there can be some false positives. So you might get a list of a thousand blocks that you need to download, and maybe of those thousand blocks only a hundred actually contain your transactions. But that's better than downloading all 666,666 plus blocks.
+
+Aaron van Wirdum: 00:09:36
+
+Yes, we made that milestone this week.
+
+Sjors Provost: 00:09:39
+
+Yeah, exactly.
+
+Aaron van Wirdum: 00:09:40
+
+That's why you remember the exact number.
+
+Sjors Provost: 00:09:42
+
+Precisely, it's very hard to remember. There are lots of trade-offs when it comes to these filters, so I think we should go into that for another episode. It's not enforced by consensus, so you're kind of trusting that the list is real. Although if you...Let's say the list gives you a bunch of nonsense results and you start downloading those blocks and you don't see what you're expecting, well then you know that whoever gave you those filters was lying to you.
```

Comment:
Here "Although if you..." is an example of mid-sentence rephrasing - eliminating it helps with readability.